### DISCUSSION TITLE
Form Builder Modals... or Actions?

### DISCUSSION THREAD

**Discussion by kurucu** [filamentphp/filament#4041] [General]
Hi all,

I've been thinking about form modals (like those present in the Table Builder) in the Form Builder. Maybe this already exists - _if so, where am I missing it please?_

But otherwise - my thoughts drove me to the idea of being able to attach Table Actions to regular livewire components, so that you could (for example) have a free button anywhere in a page that triggers an action to create a form modal, do all the stuff, and then save it at the end.

Can this be done already? Is it stupid? What do you think?

**Comment by danharrin** [comment]
filament/actions is coming in v3 - a new package that extracts the actions out of the admin panel, which can be used anywhere in a Livewire component, not just the table builder :)

**Reply by kurucu** [reply]
😮 Amazing Dan.
